package.json

```diff
@@ -10,7 +10,9 @@
     "lint": "eslint .",
     "test": "npm run lint && npm run run-tests",
     "run-tests": "lab bin/ server/ lib/ -t 80 -l -e 'development' -P spec",
-    "test-cov-html": "lab bin/ server/ lib/ -r html -o coverage.html -P spec"
+    "test-cov-html": "lab bin/ server/ lib/ -r html -o coverage.html -P spec",
```

Comment:
In other repos we try to use the gulp/grunt pattern for modifiers: `test:coverage` / `test:watch`